dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -279,13 +279,6 @@ public int authenticate(Context context, String username, String password,
 	public int[] getSpecialGroups(Context context, HttpServletRequest request)
 	{
 		try {
-			// User has not successfuly authenticated via shibboleth.
-			if ( request == null || 
-					context.getCurrentUser() == null || 
-					request.getSession().getAttribute("shib.authenticated") == null ) {
-				return new int[0];
-			}
```

Comment:
It's unclear to me why this section was removed. Shouldn't we still be checking for authentication via Shib before using the Special Groups assigned in Shib?  Is there a use case I'm missing here?